README.md

```diff
@@ -10,19 +10,19 @@ Jib is a tool for building container images for your Java applications.
 
 ## Goals
 
-* Fast - Your Java application gets broken down into multiple layers, separating dependencies from classes. Deploy your changes faster - don’t wait for Docker to rebuild your entire Java application.
+* **Fast** - Your Java application gets broken down into multiple layers, separating dependencies from classes. Deploy your changes faster - don’t wait for Docker to rebuild your entire Java application.
 
 <!--* Reproducible - Rebuilding your container image with the same contents always generates the same image. Never trigger an unnecessary update again.-->
 
-* Native - Reduce your CLI dependencies. Build your Docker image from within Maven or Gradle and push to any registry of your choice. No more writing Dockerfiles and calling docker build/push.
+* **Native** - Reduce your CLI dependencies. Build your Docker image from within Maven <!--or Gradle--> and push to any registry of your choice. No more writing Dockerfiles and calling docker build/push.
 
 ## Quickstart
 
 ### Setup
 
 In your Maven Java project, add the plugin to your `pom.xml`:
 
-```
+```xml
 <plugin>
     <groupId>com.google.com.tools</groupId>
     <artifactId>jib-maven-plugin</artifactId>
```

Comment:
I think you should put something in for `registry` `repository` and `crendential helper name` the docs looks a little strange with them empty.